rxp-hpp-android/src/main/java/com/realexpayments/hpp/ApiAdapter.java

```diff
@@ -19,13 +19,22 @@
     public static final String RETROFIT_TAG = "HPPRetrofit";
 
     public static IHPPServerAPI getAdapter(String endpoint, Map<String, String> headers) {
-        RestAdapter restAdapter = new RestAdapter.Builder()
-                .setEndpoint(endpoint)
-                .setLogLevel(RestAdapter.LogLevel.FULL)
-                .setLog(new AndroidLog(RETROFIT_TAG))
-                .setConverter(new GsonConverter(getGson()))
-                .setRequestInterceptor(getRequestInterceptor(headers))
-                .build();
+        RestAdapter restAdapter;
+        if (BuildConfig.DEBUG) {
```

Comment:
Could be resumed like:

```
        restAdapter = new RestAdapter.Builder()
                        .setEndpoint(endpoint)
                        .setConverter(new GsonConverter(getGson()))
                        .setRequestInterceptor(getRequestInterceptor(headers));

        if (BuildConfig.DEBUG) {
            restAdapter
                    .setLogLevel(RestAdapter.LogLevel.FULL)
                    .setLog(new AndroidLog(RETROFIT_TAG));
        }

        restAdapter.build();
```

Reply:
My IDE code does not recognize: BuildConfig.DEBUG, do I have do or import anything else ?